libs/atqr/domain/src/lib/challenge/challenge.entity.ts

```diff
@@ -29,30 +29,32 @@ export class Challenge {
   private _status?: ChallengeStatus;
 
   constructor(
-    private _goal: string,
-    private _supervisorName: string,
-    private _supervisorEmail: EmailAddress,
-    private _player: Player,
-    price: number,
-    deadline: Date,
-    private _paymentMethod?: PaymentMethodEntity,
-    _status: ChallengeStatus = ChallengeStatus.Ongoing,
-    private _supervisorStatus: SupervisorEnum = SupervisorEnum.notInvited
+    private _goalId: GoalId,
+    private _owner: User,
+    private _invitees: Invitee[],
+    private _deadline: Date,
+    private _pledge: number,
+    private _requiredProof?: string,
```

Comment:
you forgot to add "inviteByLink" and "owner" properties

Reply:
owner is right there, on line 33 . I was thinking inviteByLink was a function condition @rairezende